helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -39,13 +39,23 @@ spec:
         sidecar.istio.io/inject: "false"
         {{- end }}
     spec:
+      {{- if not .Values.isWindows }}
       hostNetwork: true
+      {{- end }}
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: {{ template "splunk-otel-collector.serviceAccountName" . }}
+      {{- if .Values.isWindows }}
+      nodeSelector:
+        kubernetes.io/os: windows
```

Comment:
Since we already has bunch of custom configuration requirements for Windows in README.md. Let's move this part there as well.

Reply:
For `nodeSelector` i have taken [reference](https://github.com/signalfx/signalfx-agent/blob/main/deployments/k8s/helm/signalfx-agent/templates/daemonset.yaml#L60) of smart-agent. in which value is setting based on the `isWindows` flag.
If you suggest then we can remove this part from confirmation and put it in README.md

Reply:
new version looks better